# Machine learning in bioinformatics research associate

The Structural and Functional Genomics lab at the Małopolska Centre of Biotechnology is **looking for a postdoctoral research associate to join the group**. The position is available from September 2020 with funding covered by the project until June 2023.

I have recently moved from University of California San Diego to Poland and started a group in June 2019 at the [Małopolska Centre of Biotechnology](https://mcb.uj.edu.pl/), Jagiellonian University in Kraków.  

<!-- **I am still accepting applications for this position!** -->

## The project

### Predicting microbiome changes over time and upon perturbation: applications of machine learning to microbiome research and therapeutic strategies

__Background__  
The human body forms different ecological niches for microoranisms. We host a plethora of microbes on each part of our body – different parts of our skin, our mouths; and in our body, especially in the gut. Those different niches are called microbiomes, for example the gut microbiome. In a bit over a decade the scientists described different microbiomes and learned that the microbiome is not only necessary, as it co-evolved with us, diverse and complex, but also important to health. Changes in the composition of microbes may result in an imbalance within the ecosystem of our guts called dysbiosis, which in turn may lead or contribute to disease. We already know a wide array of microbiome-disease associations, from the quite expected ones, such as obesity or gastrointestinal problems (ulcerative colitis or Crohn’s disease), to diseases presumed far removed from the gut – type-1 diabetes, allergies, depression, anxiety, neurodegenerative disorders, cancer, and others.  
    
With those significant health implications and vast amounts of data gathered over the years we are now in a position to turn the knowledge into action – learn to predict how the microbiome changes over time and upon perturbation. With this, we are going to learn how to change the microbiome in order to promote health or devise therapeutic strategies. Those strategies may involve probiotic interventions, dietary or lifestyle changes, not necessarily classical small-molecule drugs. This is no easy task as the microbiome is massive – 10s of trillions of microbial cells representing 100s of microbial species, harboring over 100 times more genes than the human genome. On top of that the microbiome is dynamic. Unlike our genome, which is fixed from birth, the microbiome changes from day to day, evolves with our diet, lifestyle and other life events.  
     
__Project__
In this project we will employ a cautious step-wise strategy to pave the way for future smart microbiome-oriented therapies. As a first step, we will construct computational models of how the microbiome changes from day-to-day. Analyzing long time-series data from several individuals we are going to construct statistical and machine learning-based models (auto-encoders) to predict those changes. Thanks to those models, we will learn what features (microbes or combinations of microbes) are important for microbiome evolution over time. We will also better learn which specific algorithms and neural network architectures work for this problem. Once we master this task, we will be ready to proceed to a more complex one.

One of the few microbiome-oriented therapeutic strategies are the fecal microbiome transplants (FMTs). Initially, they were used for the treatment of Clostridium difficile infections, but now they also find their use in the treatment of ulcerative colitis. Collecting the data from already published studies, we will predict the composition of patient’s microbiome after an FMT. This is a substantially more complicated task, so we will use previously designed models for time-series data to give us a head start (transfer learning paradigm).
    
__Future__
As a result of this project, we will learn how to predict the human gut microbiome changes over time and in response to FMTs. This will be a first step towards intelligent microbiome-oriented therapeutic strategies which give significant hope for many of the most debilitating diseases of today.

## The role
Your role is going to be to develop the machine learning algorithms to predict the microbiome composition over time and upon perturbation. You are going to work closely with a PhD student who is going to curate and analyse the metagenomic microbiome data.  

As a part of your position you are also going to be expected to contribute to the computational expertise within the group and to be proactive in solving other bioinformatic research tasks.

The position will be funded by [National Science Centre](https://www.ncn.gov.pl/?language=en) SONATA 15 project.  
  
**PLEASE NOTE:** due to [NCN regulations](https://www.ncn.gov.pl/sites/default/files/pliki/uchwaly-rady/2019/uchwala90_2019-zal1.pdf) I am __not allowed to hire postdocs who obtained their PhDs at the Jagiellonian University__. If you wish to contact NCN and let them know what you think about it, please e-mail: [kancelaria@ncn.gov.pl](mailto:kancelaria@ncn.gov.pl)

## The group

We are a young Bioinformatics group rallying around the goal of understanding understand how the microbiome works and how we can leverage it for health and wellbeing. Combinding metagenomics, data science, machine learning, structural bioinformatics and a collaborative mindset we want to develop a multi-scale understanding of the microbiome, from genes, through structures, to functions, and therapies. We also extensively collaborate with local and international research groups and companies to grow and deliver cutting-edge, open and reproducible research which would serve the community.  

## Institution

**Małopolska Centre for Biotechnology** (MCB) is a flagship biotechnology institute of Southern Poland and one of the most promising biomedical institutes in Central/Eastern Europe. It is part of the Jagiellonian University and its excellence is based on three main areas: computational biology, structural biology and plant biology. The first area, computational biology, has recently been undergoing rapid growth, with a recent start of several groups, including Bioinformatics/Transcriptomics ([Dr. Paweł Łabaj](https://scholar.google.pl/citations?hl=en&user=al6BDt8AAAAJ&view_op=list_works&sortby=pubdate)), Microbial Genomics ([Dr. Rafał Mostowy](https://mostowylab.com)), and Structural and Functional Genomics (myself). The institute will also profit from the interactions with the nearby group in Microbial Ecology and Evolution ([Dr. Piotr Łukasik](https://scholar.google.com/citations?user=nqaO1yUAAAAJ&hl=en)) and with industry ([Ardigen](https://ardigen.com)). Microbiome is a nascent field in Central/Eastern Europe and at the moment Kraków, especially MCB with 3 groups oriented around different facets of microbiome research, are rising as the leaders in this field.

MCB offers an international and collaborative environment with access to state-of-the-art infrastructure in both experimental and computational biology. MCB hosts many world-class research groups, including the Max Planck Research Group in Structural Biology ([Glatt Lab](http://glatt-lab.pl/)), the Bionanoscience and Biochemistry Group ([Heddle Lab](http://www.heddlelab.org/)), the Plant Molecular Biology Group ([Yamada Lab](https://mcb.uj.edu.pl/plant-molecular-biology-laboratory)) or the Virogenetics and Virology Group ([Pyrć Lab](http://virogenetics.info/)).


## The successful candidate will

*   a PhD degree in computer science, or a quantitative field such as biology, bioinformatics, physics or mathematics,
*   have excellent written and oral communication skills in English,
*   have _expertise_ in machine learning (esp. deep learning methods and software, e.g. TensorFlow, Keras),
*   have _documented experience_ in test-driven software development,
*   have a collaborative mindset,
*   enjoy interdisciplinary work,
*   understand the importance of communication and interactions with other group members, collaborators and Centre members.


## The ideal candidate will

*   fulfill all of the expectations of the “successful candidate”,
*   have a track record of presenting at conferences and published research,
*   have experience in data science, metagenomics, or biology,
*   have an active interest in microbiome research.


## We offer

*   competitive salary commensurate with experience,
*   opportunity to collaborate with world-class researchers from MCB and institutions such as University of California San Diego, Warsaw University,
*   scientific and professional mentorship,
*   position 100% focused on research and development (no compulsory teaching),
*   opportunity to work on an ambitious and important scientific problem with interdisciplinary applications,
*   opportunity and support to participate scientific meetings across the globe,
*   flexible working time,
*   opportunity to work (partly) remotely.


## How to inquire

Please send an email with a subject line “_Sonata postdoc_” to [work@tomaszlab.org](mailto:work@tomaszlab.org) including:

*   a cover letter including a short description of your key achievements, as well as a short explanation for why you would like to join my lab and how this position fits into your interests and career aspirations,
*   CV,
*   sample code or a link to a repositiory,
*   names, contact details and professional relationship status of 3 potential referees.  

<!--
Please include the following statement in your application: “I hereby authorize you to process my personal data included in my job application for the needs of the recruitment process (in accordance with the Act of 29 August 1997 on the protection of personal data, Dz. U. No 133, item. 883, as amended)”.
-->
### **Application deadline is Friday, July 31 2020.**

We are committed to equal opportunities and will not discriminate anyone on the basis of: nationality, age, sex, race, disability, religion, sexual orientation and pregnancy/maternity.

<!--
The recruitment will be carried out in 2 steps. Selected applicants will be interviewed after the application deadline has passed.
-->
Selected applicants will be contacted within 2 weeks of applying.
