<!--
# Graduate student in bioinformatics
-->
# Pre-doctoral fellow in bioinformatics

The Structural and Functional Genomics lab at the Małopolska Centre of Biotechnology is **looking for a pre-doctoral research assistant to join the group**. The position is available from September 2020 with funding covered by the project until June 2023.

I have recently moved from University of California San Diego to Poland and started a group in June 2019 at the [Małopolska Centre of Biotechnology](https://mcb.uj.edu.pl/), Jagiellonian University in Kraków.  


## The project

**Predicting microbiome changes over time and upon perturbation: applications of machine learning to microbiome research and therapeutic strategies**

__Background__  
The human body forms different ecological niches for microoranisms. We host a plethora of microbes on each part of our body – different parts of our skin, our mouths; and in our body, especially in the gut. Those different niches are called microbiomes, for example the gut microbiome. In a bit over a decade the scientists described different microbiomes and learned that the microbiome is not only necessary, as it co-evolved with us, diverse and complex, but also important to health. Changes in the composition of microbes may result in an imbalance within the ecosystem of our guts called dysbiosis, which in turn may lead or contribute to disease. We already know a wide array of microbiome-disease associations, from the quite expected ones, such as obesity or gastrointestinal problems (ulcerative colitis or Crohn’s disease), to diseases presumed far removed from the gut – type-1 diabetes, allergies, depression, anxiety, neurodegenerative disorders, cancer, and others.  
    
With those significant health implications and vast amounts of data gathered over the years we are now in a position to turn the knowledge into action – learn to predict how the microbiome changes over time and upon perturbation. With this, we are going to learn how to change the microbiome in order to promote health or devise therapeutic strategies. Those strategies may involve probiotic interventions, dietary or lifestyle changes, not necessarily classical small-molecule drugs. This is no easy task as the microbiome is massive – 10s of trillions of microbial cells representing 100s of microbial species, harboring over 100 times more genes than the human genome. On top of that the microbiome is dynamic. Unlike our genome, which is fixed from birth, the microbiome changes from day to day, evolves with our diet, lifestyle and other life events.  
     
__Project__
In this project we will employ a cautious step-wise strategy to pave the way for future smart microbiome-oriented therapies. As a first step, we will construct computational models of how the microbiome changes from day-to-day. Analyzing long time-series data from several individuals we are going to construct statistical and machine learning-based models (auto-encoders) to predict those changes. Thanks to those models, we will learn what features (microbes or combinations of microbes) are important for microbiome evolution over time. We will also better learn which specific algorithms and neural network architectures work for this problem. Once we master this task, we will be ready to proceed to a more complex one.

One of the few microbiome-oriented therapeutic strategies are the fecal microbiome transplants (FMTs). Initially, they were used for the treatment of Clostridium difficile infections, but now they also find their use in the treatment of ulcerative colitis. Collecting the data from already published studies, we will predict the composition of patient’s microbiome after an FMT. This is a substantially more complicated task, so we will use previously designed models for time-series data to give us a head start (transfer learning paradigm).
    
__Future__
As a result of this project, we will learn how to predict the human gut microbiome changes over time and in response to FMTs. This will be a first step towards intelligent microbiome-oriented therapeutic strategies which give significant hope for many of the most debilitating diseases of today.


## The role

The work will be focused on protein structure prediction, and structure-assisted protein function predictions using deep learning methods to understand the functional potential within the human gut microbiome. Your role will be to improve function predictions by testing various sources of structural information and determining the contributions the structures play in improving function predictions. Once the function predictions are optimized, we will use them to characterize genomes and metagenomes from the human gut microbiome.

The work will be carried out as a part of the [Microbiome Immunity Project](https://www.worldcommunitygrid.org/research/mip1/overview.do). There will also be multiple opportunities to work on projects related to the human microbiome analyses, especially in type-1 diabetes, and mental health (the gut-brain axis).

The role will be **exclusively computational but with significant implications for experimental applications**. It will involve collaborative work with high-profile international groups. **The results of the project will contribute to our understanding of the function of the human gut microbiome**, and to the design of precision medicine strategies (therapies and interventions) targeting the microbiome.

The student will be funded by the project financed from the Polish Returns grant from the [Polish National Agency for Academic Exchange](https://nawa.gov.pl/en/).


## The group

3 zdania

## Institution

**Małopolska Centre for Biotechnology** (MCB) is a flagship biotechnology institute of Southern Poland and one of the most promising biomedical institutes in Central/Eastern Europe. It is part of the Jagiellonian University and its excellence is based on three main areas: computational biology, structural biology and plant biology. The first area, computational biology, has recently been undergoing rapid growth, with the start of several groups, including the Bioinformatics/Transcriptomics group ([Dr. Paweł Łabaj](https://scholar.google.pl/citations?hl=en&user=al6BDt8AAAAJ&view_op=list_works&sortby=pubdate)), Microbial Genomics ([Dr. Rafał Mostowy](https://mostowylab.com)) group, and Structural and Functional Genomics group (myself). The institute will also profit from interactions with the nearby group in Microbial Ecology and Evolution ([Dr. Piotr Łukasik](https://scholar.google.com/citations?user=nqaO1yUAAAAJ&hl=en)) and with industry ([Ardigen](https://ardigen.com)). Microbiome is a nascent field in Central/Eastern Europe and at the moment Kraków, especially the MCB with 3 groups oriented around different facets of microbiome research, are rising as the leaders in this field.

MCB offers an international and collaborative environment with access to state-of-the-art infrastructure in both experimental and computational biology. MCB hosts many world-class research groups, including the Max Planck Research Group in Structural Biology ([Glatt Lab](http://glatt-lab.pl/)), the Bionanoscience and Biochemistry Group ([Heddle Lab](http://www.heddlelab.org/)), the Plant Molecular Biology Group ([Yamada Lab](https://mcb.uj.edu.pl/plant-molecular-biology-laboratory)) or the Virogenetics and Virology Group ([Pyrć Lab](http://virogenetics.info/)).


## The successful candidate will

*   hold an undergraduate degree in a quantitative field like biology, chemistry, physics, mathematics, computer science or similar by the time of starting the position,
*   enroll in the Doctoral School programme at the Jagiellonian University in Kraków,
*   have excellent written and oral communication skills in English,
*   be open to learning at least one programming language (esp. Python),
*   have a collaborative mindset,
*   understand the importance of communication and interactions with other group members, collaborators and Centre members.


## The ideal candidate will

*   fulfill all of the expectations of the “successful candidate”,
*   have experience in either data science, statistics, machine learning, and/or metagenomics, structural bioinformatics,
*   have documented experience in test-driven software development,
*   have an active interest in microbiome research,
*   have a track record of presenting at research conferences and/or published research.


## We offer

*   a competitive full-time PhD fellowship,
*   the opportunity to collaborate with world-class researchers from MCB and institutions such as the University of California San Diego, Warsaw University,
*   scientific mentorship,
*   the opportunity to work on an challenging and important scientific project with interdisciplinary applications,
*   the opportunity and support to participate scientific meetings across the globe,
*   flexible working time.


## How to inquire

Please send an email with a subject line “_Sonata PhD_” to [work@tomaszlab.org](mailto:work@tomaszlab.org) including:

*   a cover letter including a short description of your key achievements, as well as a short explanation for why you would like to join my lab and how this position fits with your interests and career aspirations,
*   an academic CV,
*   names, contact details and professional relationship status of 2 potential referees.

<!--
**Official job postings may be found here:**
* [in Polish](https://bip.uj.edu.pl/praca/inne-stanowiska-nauczycieli-akademickich?p_p_id=56_INSTANCE_I4gt&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&p_p_col_id=column-3&p_p_col_count=1&groupId=1384597&articleId=142939334&widok=ogloszenie)
* [in English](https://euraxess.ec.europa.eu/jobs/415156)
-->

### **Applicants will be contacted within 2 weeks of applying.**

We are committed to equal opportunities and will not discriminate anyone on the basis of: nationality, age, sex, race, disability, religion, sexual orientation and pregnancy/maternity.

<!--
Selected applicants will be interviewed after the application deadline has passed.
-->
